src/docs/sansheets/functions/index.md

```diff
@@ -1657,7 +2543,25 @@ Returns the number of transactions transferring more than 100k USD.
-| aggregation | <code>string</code> | Aggregation for the timeseries metrics. Example: "LAST"                                                                                                                                                                                           |
+| projectSlug | <code>string</code> | Name of the asset at sanbase, which can be found at the end of the URL (eg. the Santiment URL is https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).                                                                                              |
+| from        | <code>date</code>   | The starting date to fetch the data. Example: DATE(2018, 9, 20)                                                                                                                                                                                                                     |
+| to          | <code>date</code>   | The ending date to fetch the data. Example: DATE(2018, 9, 21)                                                                                                                                                                                                                       |
+| aggregation | <code>string</code> | Aggregation for the timeseries metrics. Example: "LAST"                                                                                                                                                                                                                             |
+
+## SAN_WHALE_TRANSACTION_COUNT_MULTIPLE_SLUGS
+
+##### SAN_WHALE_TRANSACTION_COUNT_MULTIPLE_SLUGS(projectSlugsList, from, to, interval) ⇒ <code>Array</code>
+
+Returns the number of transactions transferring more than 100k USD.
+
+- **Kind**: global function
+- **Returns**: <code>Array</code> - of results for multiple slugs
+the number of transactions transferring more than 100k USD.
+- **Customfunction**:
+
+| Param            | Type                | Description                                                                                                                                                                            |
+| ---------------- | ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| projectSlugsList | <code>string</code> | Comma-separated names of the assets at sanbase, which can be found at the end of the URL (eg. the Santiment URL is https://app.santiment.net/projects/santiment, https://app.santiment.net/projects/bitcoin is Bitcoin's URL, so the projectSlugList would be "santiment,bitcoin"). |
```

Comment:
@spiderjako We need to find another way to explain how to find slug. Cause such URLs don't work now.